docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+UTF-8 support in the console can be found via the _A_ variant of all Console APIs or the filesystem [**WriteFile**](https://msdn.microsoft.com/library/windows/desktop/aa365747) and [**ReadFile**](https://msdn.microsoft.com/library/windows/desktop/aa365467) methods against console handles after setting the codepage to `65001` or `CP_UTF8` with the [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods as appropriate. Setting the code pages in advance is only necessary if the machine has not chosen "Use Unicode UTF-8 for worldwide language support" in the settings for Non-Unicode applications in the Region section of the Control Panel.
+
+UTF-16 support in the console can be found with no additional configuration via the _W_ variant of all console APIs and is a more likely choice for applications already well versed in UTF-16 through communication with the `wchar_t` and _W_ variant of other Microsoft and Windows platform functions and products.
+
+## Recommendations
+
+For all new and ongoing development on Windows, virtual terminal sequences are recommended as the way of interacting with the terminal. This will converge Windows command-line client applications with the style of application programming on all other platforms.
+
+A limited subset of Windows Console APIs are still necessary to establish the initial environment as the Windows platform still differs from others in process, signal, device, and encoding handling:
+
+- The standard handles to a process will still be controlled with **[GetStdHandle](getstdhandle.md)** and **[SetStdHandle](setsetdhandle.md)**.
+- Configuration of the console modes on a handle to opt in to Virtual Terminal Sequence support will be handled with **[GetConsoleMode](getconsolemode.md)** and **[SetConsoleMode](setconsolemode.md)**.
+- Declaration of code page or UTF-8 support is conducted with [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods.
+- Some level of overall process management may be required with the [**AllocConsole**](allocconsole.md), [**AttachConsole**](attachconsole.md) and [**FreeConsole**](freeconsole.md) to join or leave a console device session.
+- Signals and signal handling will continue to be conducted with [**SetConsoleCtrlHandler**](setconsolectrlhandler.md), [**HandlerRoutine**](handlerroutine.md), and [**GenerateConsoleCtrlEvent**](generateconsolectrlevent.md).
+- Communication with the console device handles can be conducted with [**WriteConsole**](writeconsole.md), [**ReadConsole**](readconsole.md), [**WriteFile**](https://msdn.microsoft.com/library/windows/desktop/aa365747) and [**ReadFile**](https://msdn.microsoft.com/library/windows/desktop/aa365467).

```

Comment:
We do *not* have a good story for learning about window size changes using only ReadFile ☹️

Reply:
added note